zos_concepts/data_transfer/archive_copy_unarchive_restore/archive-fetch.yml

```diff
@@ -0,0 +1,99 @@
+###############################################################################
+# © Copyright IBM Corporation 2023
+###############################################################################
+
+###############################################################################
+# This playbook demonstrates how to use zos_archive to archive a set of data sets  in z/OS
+# and zos_fetch to get the archive to the controller using Red Hat Ansible Certified Content for IBM Z.
+#
+# Usage:
+#  ansible-playbook -i <inventory> <playbook>
+#
+# Example:
+#  ansible-playbook -i inventories archive-fetch.yml
+#
+#
+# Requirements:
+#   IBM z/OS core collection 1.7.0 or later.
+#   Ansible Community General Collection
+#
+#   The ansible.cfg file for this playbook contains the setting stdout_callback = unixy
+#   which changes the format of the playbook output. It has been set to better demonstrate
+#   the effects of whitespace (as compared to the default output formatting).
+#   The unixy callback is part of the community.general package (which is maintained
+#   by the Ansible community).#
+#
+###############################################################################
+
+- hosts: source
+  collections :
+    - "ibm.ibm_zos_core"
+  gather_facts: false
+  environment: '{{ environment_vars }}'
+
+  tasks:
+    - name: Raise error if data_sets_to_archive is not set
+      ansible.builtin.fail:
+        msg: Missing required variable 'data_sets_to_archive'.
+      failed_when: data_sets_to_archive is not defined
+
+    #########################################################################
+    # This first section uses dls -s to get the space attributes of the data
+    # set list, then executes filter_plugins/total_size.py to filter the
+    # response and convert into cyl space type size.
+    #########################################################################
+    - name: Get attributes for data sets
+      ansible.builtin.command: dls -s {{ data_sets_to_archive | join(' ') }}
+      register: response
+
+    - ansible.builtin.set_fact:
+        size_in_cyls: "{{ response | total_size }}"
+
+    #########################################################################
+    # This section executes a DUMP using ADRDSSU and then archives it using terse
+    # format in one task using zos_archive.
+    # Options explanation:
+    # - format object contains the format type in name and the options inside
```

Comment:
Modified the "remove" option since is not under format_options:

```    
   # The format option contains a number of sub-options including 'name' to
    # specify the format type, 'format_options' which contains two sub-options
    # terse_pack to specify SPACK or PACK and 'use_addrsu' to specify whether
    # to execute a previous step to create and archive a dump data set.
    # The 'remove' option which removes data sets from the systems upon a
    # successful archive operation.
    # The dest_data_set option is used to specify the desired size attributes
    # of the archive.
    ```